opentelemetry-semantic-conventions/scripts/templates/semantic_attributes.rs.j2

```diff
@@ -14,7 +14,9 @@
 {%- set attribute = (attrs | selectattr('deprecated', 'none') | first) %}
 {%- set attribute = attribute if attribute else (attrs | first) %}
 {%- if conventions != 'attribute' %}
+{%- if not attribute.deprecated %}

```

Comment:
conditional check to ensure that the deprecated constants only defined with `#[deprecated]` annotation, and not exported.